js/app/controllers/editorcontroller.js

```diff
@@ -121,6 +121,14 @@ app.controller('EditorController', ['$scope', 'TimezoneService', 'AutoCompletion
 			});
 		};
 
+		$scope.keypress = function(event) {
+			var code = event.keyCode || event.which;
+			if(event.ctrlKey===true && code === 13) {
```

Comment:
Can you please use `event.metaKey`. Then it will also work on Macs. :)

https://api.jquery.com/event.metakey/